examples/xonotic/cloudbuild.yaml

```diff
@@ -0,0 +1,51 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  
+  #
+  # Creates the initial make + docker build platform
+  #
+  - name: "ubuntu"
+    args:
+    - "bash"
+    - "-c"
+    - "echo 'FROM gcr.io/cloud-builders/docker\nRUN apt-get install make\nENTRYPOINT [\"/usr/bin/make\"]' > Dockerfile.build"
```

Comment:
Non blocking, but this could actually now use the new `script` element in Cloud Build:

https://cloud.google.com/build/docs/configuring-builds/run-bash-scripts

Reply:
script block added in the steps